pages/createRecipe/create-recipe.js

```diff
@@ -32,8 +31,16 @@ async function fetchRecipeAdmin(ingredients){
     
     const recipeRequest = {
```

Comment:
Consider having this logic in the backend in the future and only deliver a clean and simple json file to the frontend.